numpy/distutils/system_info.py

```diff
@@ -92,20 +92,20 @@
 search_static_first = 0
 
 [fftw]
-fftw_libs = rfftw, fftw
-fftw_opt_libs = rfftw_threaded, fftw_threaded
-# if the above aren't found, look for {s,d}fftw_libs and {s,d}fftw_opt_libs
```

Comment:
Why did you remove this comment as part of this PR?

Reply:
same as for previous comment